ext/date/config0.m4

```diff
@@ -8,15 +8,24 @@ AX_CHECK_COMPILE_FLAG([-Wno-implicit-fallthrough],
   [PHP_DATE_CFLAGS="$PHP_DATE_CFLAGS -Wno-implicit-fallthrough"],,
   [-Werror])
 
-PHP_DATE_CFLAGS="$PHP_DATE_CFLAGS -I@ext_builddir@/lib -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -DHAVE_TIMELIB_CONFIG_H=1"
+PHP_DATE_CFLAGS="$PHP_DATE_CFLAGS -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -DHAVE_TIMELIB_CONFIG_H=1"
+PHP_TIMELIB_CFLAGS="$PHP_DATE_CFLAGS"
+PHP_DATE_CFLAGS="$PHP_DATE_CFLAGS -I@ext_builddir@/lib"
```

Comment:
I'm sure this is fine, but of course `-DZEND_ENABLE_STATIC_TSRMC_CACHE=1` is meaningless in the `PHP_TIMELIB_CFLAGS` definition?

Reply:
Yes you are right, I'll move that one too.
Furthermore, I'll apply cmb's suggestion of applying `fwrapv` to Clang on Windows too.